core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -66,6 +77,29 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     }
     sc = new SparkContext(conf)
     taskScheduler = new TaskSchedulerImpl(sc)
+    setupHelper()
+  }
+
+  def setupSchedulerWithMockTsm(): TaskSchedulerImpl = {
```

Comment:
yeah the important part here is actually the mock blacklist, not the spied tsm -- I think that name was leftover from long ago.  I've updated.